lib/model/message_list.dart

```diff
@@ -86,6 +88,72 @@ class MessageListView extends ChangeNotifier {
     notifyListeners();
   }
 
+  _applyChangesToMessage(UpdateMessageEvent event, Message message) {
+    //In earlier server versions, omitting the userId indicates that this is a rendering-only update.
+    //That means this change was initiated by the server, not the user.
```

Comment:
👍 

Does Dart have an auto-formatter? Looks like Prettier doesn't support Dart, which is too bad.

Reply:
There is a Dart autoformatter, but (like the Flutter repo itself), we don't use it. Some discussion here: https://github.com/zulip/zulip-flutter/issues/229#issuecomment-1642807019

For wrapping prose, though, I believe the auto-formatter wouldn't intervene anyway. But your IDE should have its own functionality to wrap a single paragraph or a selected region.

Reply:
For VS Code in particular (since I think you mentioned that's what you're using), I use the extension `stkb.rewrap`, and then it's Alt+Q to wrap a paragraph. I installed that extension some years ago for working in the zulip-mobile codebase, which is JavaScript and uses Prettier — Prettier doesn't seem to rewrap prose in comments either — but it works just as well for Dart.

(It's pretty reasonable for auto-formatters like Prettier and dartfmt to stay away from attempting to rewrap comments, because it's common for comments to have fragments of code or other things that aren't meant to be wrapped like prose. But for providing the developer a key shortcut to manually say to wrap a given paragraph or passage, there's no such concern.)